Library/Homebrew/language/node.rb

```diff
@@ -0,0 +1,32 @@
+module Language
+  module Node
+    def self.npm_cache_config
+      "cache=#{HOMEBREW_CACHE}/npm_cache\n"
+    end
+
+    def self.setup_npm_environment
+      # explicitly use our npm and node-gyp executables instead of the user
+      # managed ones in HOMEBREW_PREFIX/lib/node_modules which might be broken
+      ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/npm/bin"
+      # explicitly set npm's cache path to HOMEBREW_CACHE/npm_cache to fix
+      # issues caused by overriding $HOME (long build times, high disk usage)
+      # https://github.com/Homebrew/brew/pull/37#issuecomment-208840366
+      npmrc = Pathname.new("#{ENV["HOME"]}/.npmrc")
+      npmrc.write npm_cache_config unless npmrc.exist?
+    end
+
+    def self.std_npm_install_args(libexec)
+      setup_npm_environment
+      # tell npm to not install .brew_home by adding it to .npmignore
+      open('.npmignore', 'a') { |f| f.write( "\n.brew_home\n") }
```

Comment:
@chrmoritz Thanks for the comment! Then ignore my suggestions and just fix the quotes. :smiley_cat: